__tests__/__snapshots__/main.test.ts.snap

```diff
@@ -123,36 +123,18 @@ exports[`displays module information when files are added/removed/changed 1`] =
 <details>
 <summary>**Changeset**</summary>
 
-File | Δ
----- | -
-  \`src/client/this/file/has/a/long/name/so-that-it-will-hopefully-wrap.tsx\` | 🆕 +2.72 KB
-\`src/client/this-file-was-added.ts\` | 🆕 +1.46 KB
-\`src/client/this-is-another-file-with-a-long-name-to-test-the-table-sizing.tsx\` | 🆕 +1.04 KB
-\`src/client/helpers/this-file-is-much-larger.ts\` | 📈 +11.66 KB (+865.77%)
-\`src/client/this-file-grew-larger.tsx\` | 📈 +200 B (+35.59%)
-\`src/client/helpers/this-file-grew-smaller.ts\` | 📉 -200 B (-7.94%)
-\`src/client/this-file-is-much-smaller.tsx\` | 📉 -11.66 KB (-99.92%)
-\`src/client/routes.tsx\` | 🔥 -2.72 KB (-100%)
-\`src/client/this-file-will-be-deleted.ts\` | 🔥 -1.46 KB (-100%)
-\`src/client/render-memex.tsx\` | 🔥 -1.04 KB (-100%)
-
-<details>
-<summary>View individual file sizes</summary>
-
-File | Old | New | Δ
----- | --- | --- | -
-  \`src/client/this/file/has/a/long/name/so-that-it-will-hopefully-wrap.tsx\` | 0 B | 2.72 KB | 🆕 +2.72 KB
-\`src/client/this-file-was-added.ts\` | 0 B | 1.46 KB | 🆕 +1.46 KB
-\`src/client/this-is-another-file-with-a-long-name-to-test-the-table-sizing.tsx\` | 0 B | 1.04 KB | 🆕 +1.04 KB
-\`src/client/helpers/this-file-is-much-larger.ts\` | 1.35 KB | 13.01 KB | 📈 +11.66 KB (+865.77%)
-\`src/client/this-file-grew-larger.tsx\` | 562 B | 762 B | 📈 +200 B (+35.59%)
-\`src/client/helpers/this-file-grew-smaller.ts\` | 2.46 KB | 2.26 KB | 📉 -200 B (-7.94%)
-\`src/client/this-file-is-much-smaller.tsx\` | 11.67 KB | 10 B | 📉 -11.66 KB (-99.92%)
-\`src/client/routes.tsx\` | 2.72 KB | 0 B | 🔥 -2.72 KB (-100%)
-\`src/client/this-file-will-be-deleted.ts\` | 1.46 KB | 0 B | 🔥 -1.46 KB (-100%)
-\`src/client/render-memex.tsx\` | 1.04 KB | 0 B | 🔥 -1.04 KB (-100%)
-
-</details>
```

Comment:
this ouput was mostly duplicative, so just inlined some detail above to save on characters